site/_posts/2023-02-03-single-sign-on-laravel-fusionauth.md

```diff
@@ -0,0 +1,339 @@
+
+SSO helps reduce password fatigue by requiring users to only remember one password and username for all the applications managed through the SSO feature. This also reduces the number of support tickets created for your IT team when a user inevitably forgets their password. In addition, SSO minimizes the number of times you have to key-in security credentials, limiting exposure to security issues like keystroke probing and exposure. Security is also enhanced by SSO because you can implement additional functionality such as MFA or anomalous behavior detection at the identity provider without adding any complexity to your application.
+
+In this tutorial, you'll learn how to design and implement SSO using [Laravel](https://laravel.com), a popular PHP-based web framework and FusionAuth as the OIDC provider. Any other OIDC compatible authentication server should work as well.
+
+## Implementing SSO in a Laravel web app
+
+As previously stated, in this tutorial, you'll be shown how to implement SSO in a Laravel web app. The Laravel demo application is integrated with FusionAuth, an authentication and authorization platform, and [Laravel Socialite](https://laravel.com/docs/9.x/socialite), its official solution to authenticate users with OAuth providers.
+
+Before you begin, you'll need the following:
+
+* A Linux machine. The step-by-step instructions in this article are based on a [CentOS Linux machine](https://www.centos.org). If you want to work on a different OS, check out this [setup guide](https://fusionauth.io/docs/v1/tech/5-minute-setup-guide) for more information.
+* [Docker Engine](https://docs.docker.com/engine/) and [Docker Compose](https://docs.docker.com/compose/).
+* Experience with a Laravel framework and application development.
+* The [Git client tool](https://git-scm.com/downloads).
+
+## Clone the GitHub repository
+
+To begin, clone the demo project source code for this project from this [GitHub repository](https://github.com/FusionAuth/fusionauth-example-laravel-single-sign-on) using the following command:
+
+```bash
+git clone https://github.com/FusionAuth/fusionauth-example-laravel-single-sign-on
+```
+
+## Installing FusionAuth
+
+Get through the [5-Minute Setup Guide](https://fusionauth.io/docs/v1/tech/5-minute-setup-guide) to install FusionAuth in your machine. You have a few ways of doing so, but we recommend the [Docker](https://fusionauth.io/docs/v1/tech/getting-started/5-minute-docker) approach.
```

Comment:
Let's not recommend folks go through the 5 minute setup guide, which walks through a node application and could be confusing. Let's assume folks have docker since we said so in the prerequisites.

Let's just show the docker compose instructions and then remove the maintenance mode section.

Here are the docker-compose instructions: 

```
curl -o docker-compose.yml https://raw.githubusercontent.com/FusionAuth/fusionauth-containers/master/docker/fusionauth/docker-compose.yml
curl -o .env https://raw.githubusercontent.com/FusionAuth/fusionauth-containers/master/docker/fusionauth/.env
```